### ISSUE TITLE
ImportError: DLL load failed

### ISSUE THREAD

**Issue by weypro** [open-mmlab/mmcv#1862]
When I run the following command on Windows10,
```
python -c "from mmcv.utils import collect_env; print(collect_env())"
```
the error occurs.

```
'tail' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
'gcc' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
Traceback (most recent call last):
  File "C:\Anaconda3\envs\mmdtestenv\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Anaconda3\envs\mmdtestenv\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\Users\adminuser\.vscode\extensions\ms-python.python-2022.4.1\pythonFiles\lib\python\debugpy\__main__.py", line 45, in <module>
    cli.main()
  File "c:\Users\adminuser\.vscode\extensions\ms-python.python-2022.4.1\pythonFiles\lib\python\debugpy/..\debugpy\server\cli.py", line 444, in main
    run()
  File "c:\Users\adminuser\.vscode\extensions\ms-python.python-2022.4.1\pythonFiles\lib\python\debugpy/..\debugpy\server\cli.py", line 285, in run_file
    runpy.run_path(target_as_str, run_name=compat.force_str("__main__"))
  File "C:\Anaconda3\envs\mmdtestenv\lib\runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "C:\Anaconda3\envs\mmdtestenv\lib\runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "C:\Anaconda3\envs\mmdtestenv\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "d:\projects\pythonml\envtest\mdtest.py", line 3, in <module>
    print(collect_env())
  File "C:\Anaconda3\envs\mmdtestenv\lib\site-packages\mmcv\utils\env.py", line 87, in collect_env      
    from mmcv.ops import get_compiler_version, get_compiling_cuda_version
  File "C:\Anaconda3\envs\mmdtestenv\lib\site-packages\mmcv\ops\__init__.py", line 2, in <module>       
    from .active_rotated_filter import active_rotated_filter
  File "C:\Anaconda3\envs\mmdtestenv\lib\site-packages\mmcv\ops\active_rotated_filter.py", line 10, in <module>
    ['active_rotated_filter_forward', 'active_rotated_filter_backward'])
  File "C:\Anaconda3\envs\mmdtestenv\lib\site-packages\mmcv\utils\ext_loader.py", line 13, in load_ext  
    ext = importlib.import_module('mmcv.' + name)
  File "C:\Anaconda3\envs\mmdtestenv\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: DLL load failed: 找不到指定的程序。
```

My Environment:
torch 1.10.0
torchvision 0.11.0
python 3.7.13

Previous Commands:
```
pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.10.0/index.html
pip install mmdet==2.23.0
```

**Comment by zhouzaida** [comment]
Hi @weypro , thanks for your report. Could you provide the output of the following command?

```bash
pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.10.0/index.html
```

**Comment by teamwong111** [comment]
@weypro, this bug should have been solved, more details in #1789, you can fix it according to #1789 or wait two weeks for the next released version.

**Comment by weypro** [comment]
OK. Thank you. @teamwong111 @zhouzaida
